app/airq/api.py

```diff
@@ -35,29 +36,44 @@
 from airq.tasks import bulk_send
 
 
+SUPPORTED_LANGUAGES = ["en", "es"]
+
+
 def healthcheck() -> str:
     return "OK"
 
 
+def _get_supported_locale(locale: str) -> str:
+    if locale in SUPPORTED_LANGUAGES:
+        return locale
+    return "en"
+
+
 @csrf.exempt
-def sms_reply() -> str:
+def sms_reply(locale: str) -> str:
+    supported_locale = _get_supported_locale(locale)
+    g.locale = supported_locale
     resp = MessagingResponse()
     zipcode = request.values.get("Body", "").strip()
     phone_number = request.values.get("From", "").strip()
     message = commands.handle_command(
-        zipcode, phone_number, ClientIdentifierType.PHONE_NUMBER
+        zipcode, phone_number, ClientIdentifierType.PHONE_NUMBER, supported_locale
```

Comment:
Instead of passing this in, you could retrieve it from `g` in the command handler... but this is more explicit. +1

Reply:
+1